app/src/main/java/se/studieresan/studs/events/presenters/EventDetailPresenter.kt

```diff
@@ -0,0 +1,30 @@
+package se.studieresan.studs.events.presenters
+
+import se.studieresan.studs.data.Address
+import se.studieresan.studs.data.Event
+import se.studieresan.studs.events.contracts.EventDetailContract
+
+class EventDetailPresenter(
+        view: EventDetailContract.View,
+        private val event: Event
+) : EventDetailContract.Presenter {
+    private var view: EventDetailContract.View? = view
+
+    override fun didPressCompanyLocation() = view!!.openGoogleMapsNavigation(Address( event.location))
```

Comment:
Is this extra space in the argument some Kotlin thing? :thinking:

Reply:
Nope!